### ISSUE TITLE
Monitor scale change popup despite auto dpi enabled

### ISSUE THREAD

**Issue by jennirinker** [spyder-ide/spyder#13064]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description

When disconnecting/connecting to external monitor, Spyder always throws the "monitor scale change detected" popup, even though I have made sure to check the "Enable auto dpi scaling" option in preferences.

### What steps reproduce the problem?

1. Have spyder running
2. Connect or disconnect from external monitor

### What is the expected output? What do you see instead?

I assumed that if I checked "enable auto dpi scaling" I would not see this warning again.

## Versions

* Spyder version: 4.1.3
* Python version: 3.7.2
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: Windows 10 Enterprise

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.15.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.0.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.3 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.4 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.4 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.0.2 (OK)
numpy >=1.7                    :  1.15.4 (OK)
pandas >=0.13.1                :  0.24.0 (OK)
scipy >=0.17.0                 :  1.2.0 (OK)
sympy >=0.7.3                  :  None (OK)
```

**Comment by juanis2112** [comment]
Hi @jennirinker, that's actually the desired behavior because Spyder can only change the dpi scaling if it restarts so "enable auto dpi scaling" will just make sure to load the dpi according to your monitor, once spyder restarts.

**Comment by juanis2112** [comment]
Hi @jennirinker thanks for reporting this. We changed this behavior so that it only shows when dpi scaling is changed. It will be available in our next release 4.1.4!

**Comment by jennirinker** [comment]
Lovely! cheers.
